x-pack/plugins/secops/public/components/app_settings/app_settings_popover.tsx

```diff
@@ -49,12 +40,6 @@ export const AppSettingsPopover = pure<Props>(({ showPopover, onClick, onClose }
           {i18n.ADD_DATA}
         </EuiButton>
       </EuiFlexItem>
-      <EuiFlexItem>
-        <Title size="xs">
-          <TitleText>{i18n.THEME}</TitleText>
-        </Title>
-        <ThemeSwitcher />
```

Comment:
Theme switching now happens via `Kibana > Advanced Settings > theme:darkMode` and is shared across users of the workspace. Nice to be using the cooked in settings now :)